connectors/s3-connector/src/main/scala/zio/connect/s3/package.scala

```diff
@@ -0,0 +1,48 @@
+package zio.connect
+
+import zio.Trace
+import zio.connect.s3.S3Connector.{CopyObject, S3Exception}
+import zio.stream.{ZSink, ZStream}
+
+package object s3 {
+
+  def copyObject(implicit trace: Trace): ZSink[S3Connector, S3Exception, CopyObject, CopyObject, Unit] =
+    ZSink.environmentWithSink(_.get.copyObject)
```

Comment:
I think these `environmentWithXYZ` functions have a corresponding `serviceWithXYZ` version which gives you the service directly (not wrapped in `ZEnvironment` which is a bit cleaner - no need to call `.get` everywhere.